app/hackathon_variables.py

```diff
@@ -35,7 +35,7 @@
 HACKATHON_GITHUB_REPO = 'https://github.com/hackupc/registration/'
 
 # (OPTIONAL) Applications deadline
-HACKATHON_APP_DEADLINE = timezone.datetime(2019, 11, 1, 0, 0, tzinfo=timezone.pytz.timezone(TIME_ZONE))
+HACKATHON_APP_DEADLINE = timezone.datetime(2019, 10, 4, 23, 59, tzinfo=timezone.pytz.timezone(TIME_ZONE))
 # (OPTIONAL) When to arrive at the hackathon
 HACKATHON_ARRIVE = 'Registration opens at 3:00PM and closes at 6:00PM on Friday October 19th, ' \
```

Comment:
Shouldn't the registration be "Friday October 11th"?

Reply:
Registration closes a week before the event. That is Friday October 4th. 

Thanks for noticing though!

Reply:
Oh, actually, you are right. I though you were talking about the deadline! Thanks for noticing! updating it now!

Reply:
Update in #71  thanks for noticing!

Reply:
@casassg Not a big deal, you're welcome!!